packages/expo-router/CHANGELOG.md

```diff
@@ -6,10 +6,13 @@
 
 ### 🎉 New features
 
+- Add `router.navigate` and `navigate` to `useRouter` ([#24600](https://github.com/expo/expo/pull/24600) by [@marklawlor](https://github.com/marklawlor))
```

Comment:
Rename to something like `pushOrPop` to show that it's a superset of the push behavior.